extensions/mssql/yarn.lock

```diff
@@ -2,6 +2,186 @@
 # yarn lockfile v1
 
 
+"@babel/code-frame@^7.10.1":
```

Comment:
why are there so many differences in this yarn.lock file

Reply:
So for some reason the yarn.lock was referencing the 1.0.5 version of vscodetestcover - which was before my big refactor to use the istanbul libs directly. So these changes are from all that